client/context/broadcast.go

```diff
@@ -131,6 +131,10 @@ func (ctx CLIContext) broadcastTxCommit(txBytes []byte) (*ctypes.ResultBroadcast
 		return res, err
 	}
 
+	if ctx.ResponseHandler != nil {
```

Comment:
I can't quite get this. What is a response handler supposed to do? Just prettify the output? And why should a cli command want to modify `ResultBroadcastTxCommit` returned by `broadcast`?

Reply:
Yes I updated the PR with the simpler type signature. The commit is below